package.json

```diff
@@ -26,7 +26,6 @@
     "checkchange": "beachball check --changehint \"Run 'yarn change' to generate a change file\"",
     "depcheck": "rnx-dep-check && lage depcheck",
     "lint": "lage lint",
-    "postinstall": "node ./scripts/postinstall.js",
```

Comment:
Why is this removed?

Reply:
The script is deleted 😄 
![image](https://user-images.githubusercontent.com/4123478/126012569-861e3d92-9924-492d-83c1-eabd26c6721a.png)

Reply:
Ah, it's functionality moved upstream. Thanks!